bob/src/cluster/quorum.rs

```diff
@@ -77,6 +78,50 @@ impl Quorum {
         }
     }
 
+    async fn delete_at_least(&self, key: BobKey, with_aliens: bool) -> Result<(), Error> {
+        debug!("DELETE[{}] ~~~DELETE LOCAL NODE FIRST~~~", key);
+        let mut local_put_ok = 0_usize;
+        let mut remote_ok_count = 0_usize;
+        let mut at_least = self.quorum;
```

Comment:
`delete` should be performed on all node, containing data with specified `key`. And if `with_aliens` is true, then on all node in cluster